app/internal/page/donor/remove_person_to_notify.go

```diff
@@ -42,32 +42,21 @@ func RemovePersonToNotify(logger Logger, tmpl template.Template, lpaStore LpaSto
 			data.Form = readRemovePersonToNotifyForm(r)
 			data.Errors = data.Form.Validate()
 
-			if data.Form.RemovePersonToNotify == "yes" && data.Errors.None() {
-				lpa.PeopleToNotify.Delete(attorney)
```

Comment:
Looks like some copy paste var naming has come over from remove attorneys here (although not as part of this ticket)